sdk/spring-sdk-testkit/src/main/java/kalix/springsdk/testkit/ActionTestkit.java

```diff
@@ -77,14 +77,14 @@ public <R> ActionResult<R> call(Function<A, Action.Effect<R>> func) {
 
   /**
    * The {@code stream} method can be used to simulate a streamed call to the Action. The passed java lambda should
-   * return a Flux<Action.Effect>. The Flux<Effect> is interpreted into an Flux<ActionResult> that can be used in
+   * return a {@code Flux<Action.Effect>}. The {@code Flux<Action.Effect>} is interpreted into an {@code Flux<ActionResult>} that can be used in
```

Comment:
On line 79 we are still saying `stream`, should be `{@code streamedCall}`.

I can't send a GH suggestion because line is not in the diff. 😞

Reply:
right :) on sec

Reply:
done now